source/helpers/UrlHelper.js

```diff
@@ -47,16 +48,21 @@ const encodeQueryData = (queryParams) => {
  * @param {string} endpoint
  * @param {obj} params
  */
-export const buildServiceUrl = (endpoint = '', params = {}) => {
-  let queryParams = { apikey: env.MITTHELSINGBORG_IO_APIKEY || '' };
+export const buildServiceUrl = async (endpoint = '', params = {}) => {
+  const appEnv = await StorageService.getData(APP_ENV_KEY);
+  const devMode = appEnv === 'development';
+  const apiUrl = devMode ? env.MITTHELSINGBORG_IO_DEV : env.MITTHELSINGBORG_IO;
+  const apiKey = devMode ? env.MITTHELSINGBORG_IO_DEV_APIKEY : env.MITTHELSINGBORG_IO_APIKEY;
+
+  let queryParams = { apiKey };
   // Concatenate params
   queryParams = { ...params, ...queryParams };
```

Comment:
This looks weird, why not just 
```suggestion
const queryParams = { ...params, apiKey };
```

Reply:
That was weird. Fixed!